server/config/config.go

```diff
@@ -199,6 +199,8 @@ const (
 	defaultDisableErrorVerbose = true
 )
 
+var defaultRuntimeServices = []string{""}
```

Comment:
why not use `[]string{}`?

Reply:
Maybe `var defaultRuntimeServices []string` is more recommended.

Reply:
em, that's is different. `var defaultRuntimeServices []string` creates a `nil` variable, which will marshal to `null`

Reply:
get